torchvision/ops/roi_align.py

```diff
@@ -12,6 +14,24 @@
 from ._utils import check_roi_boxes_shape, convert_boxes_to_roi_format
 
 
+def lazy_compile(**compile_kwargs):
+    """Lazily wrap a function with torch.compile on the first call
+
+    This avoids eagerly importing dynamo.
```

Comment:
Am I understanding this correctly?

```suggestion
    This avoids eagerly compiling a function at import time.
```

Reply:
Nope. Even with torch.compile at top level it isn't compiled until you call it the first time. But importing dynamo has undesirable side effects for eager mode only users so it's best not to do it.